### ISSUE TITLE
bug: ionic.bundle.min.js broken in master

### ISSUE THREAD

**Issue by bymodude** [driftyco/ionic#4367]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>was trying to update ionic master to latest to get the ios9 fixes:
the minified ionic.bundle.min.js from release: v1.1.0-nightly-1558 is broken and results in app not starting
worked fine in v1.1.0-nightly-1533
maybe because of this? https://github.com/driftyco/ionic/commit/c145945f5b5dfb6171677204225efa2d8592fe0b#commitcomment-13235350
```
Uncaught Error: [$injector:modulerr] http://errors.angularjs.org/1.4.3/$injector/modulerr?p0=testapp&p1=Error…ttp%3A%2F%2Flocalhost%3A8100%2Fjs%2Fionic%2Fionic.bundle.min.js%3A50%3A339)
(anonymous function) @ ionic.bundle.min.js:37
(anonymous function) @ ionic.bundle.min.js:69
m @ ionic.bundle.min.js:38
g @ ionic.bundle.min.js:68
eb @ ionic.bundle.min.js:71
d @ ionic.bundle.min.js:50
Ac @ ionic.bundle.min.js:51
(anonymous function) @ (index):287
n.Callbacks.j @ jquery.js:3099
n.Callbacks.k.fireWith @ jquery.js:3211
n.extend.ready @ jquery.js:3417
I @ jquery.js:3433
```
</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @bymodude!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4367). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4367)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by perrygovier** [comment]
Hey @bymodude, that's entirely possible. We don't see that in our example apps, can you share an example that reproduces this?

**Comment by bymodude** [comment]
it seems the fix would be simply this: https://gist.github.com/IgorMinar/863acd413e3925bf282c#gistcomment-1574927

**Comment by perrygovier** [comment]
'doh!

**Comment by perrygovier** [comment]
fixed in 861297a99a82c60921a6faa2dffd6cbe573c2564
